vector/src/main/java/im/vector/app/features/settings/devices/v2/list/SessionInfoView.kt

```diff
@@ -187,6 +189,7 @@ class SessionInfoView @JvmOverloads constructor(
             views.sessionInfoLastActivityTextView.isGone = true
         }
         views.sessionInfoLastIPAddressTextView.setTextOrHide(deviceInfo.lastSeenIp?.takeIf { isLastSeenDetailsVisible })
```

Comment:
I think we could write instead:
views.sessionInfoLastIPAddressTextView.setTextOrHide(deviceInfo.lastSeenIp?.takeIf { isLastSeenDetailsVisible && isShowingIpAddress })